README.md

```diff
@@ -44,6 +45,14 @@ We plan to remove the need for this work-around via [OpenSearch#581](https://git
 
 To upgrade your existing plugins to work with OpenSearch, see [UPGRADING](./UPGRADING.md).
 
+### Installing Plugins in OpenSearch
```

Comment:
I think this is actually a bit unclear.

1) Are we telling people to assemble, extract and run OpenSearch? If so, this should say "Assemble, extract and run OpenSearch" and provide an example.

```
~/OpenSearch (main)> ./gradlew build
~/OpenSearch (main)> ./gradlew :distribution:archives:linux-tar:assemble
~/OpenSearch (main)> tar vfxz distribution/archives/linux-tar/build/distributions/opensearch-1.0.0-SNAPSHOT-linux-x64.tar.gz
~/OpenSearch (main)> cd opensearch-1.0.0-SNAPSHOT
~/OpenSearch/opensearch-1.0.0-SNAPSHOT (main)> ./bin/opensearch
```

Then, to build and install a plugin, for example job-scheduler:

```
~/job-scheduler (main)>  ./gradlew  assemble -Dbuild.version_qualifier=beta1 -Dbuild.snapshot=false
```

```
~/OpenSearch/opensearch-1.0.0-SNAPSHOT (main)> ./bin/opensearch-plugin install file://<your path>/build/distributions/opensearch-job-scheduler-1.0.0.0-beta1.zip
-> Installing file://.../build/distributions/opensearch-job-scheduler-1.0.0.0-beta1.zip
-> Downloading file://.../build/distributions/opensearch-job-scheduler-1.0.0.0-beta1.zip
[=================================================] 100%   
-> Installed opensearch-job-scheduler
```

2) If we're telling people to download a distribution, we should say from where and also provide a working example.

Probably both?

Reply:
Futhermore, many plugins like anomaly-detection are not useful without dashboards if I am not mistaken. Let's explain how to get that onto here as well?

Reply:
Thats a great point. I started off to answer how can we install plugins but to work with a specific version I didnt think about that. But I definitely agree that would help. 

I'll take some of these changes and add them to the PR.

Reply:
Not really, plugins do have APIs but they are way more easier to have Dashboards plugins to use a UI. 
But I do agree, will have instructions to have both the plugins installed.